python/kfserving/kfserving/transformer.py

```diff
@@ -0,0 +1,61 @@
+import logging
+from typing import List
+import requests
+import numpy as np
+
+from kfserving.server import Protocol
+from kfserving.server import KFModel
+from kfserving.server import KFSERVER_LOGLEVEL
+import kfserving.protocols.seldon_http as seldon
+from kfserving.protocols.seldon_http import SeldonRequestHandler
+
+logging.basicConfig(level=KFSERVER_LOGLEVEL)
+
+
+class Transformer(KFModel):
+    def __init__(self, name: str,
+                 predict_url: str,
+                 protocol: Protocol):
+        super().__init__(name)
+        self.predict_url = predict_url
+        self.protocol = protocol
+
+    def load(self):
+        if not self.ready:
+            logging.info("Loading transformer")
+            self.ready = True
+
+    def preprocess(self, inputs: List) -> List:
+        return inputs
+
+    def postprocess(self, inputs: List) -> List:
+        return inputs
+
+    def predict(self, inputs: List) -> List:
```

Comment:
did you consider creating a KFServing dataplane client instead?

Reply:
moved this to the protocol request handler